Gemfile.lock

```diff
@@ -262,10 +262,6 @@ GEM
     nokogiri (1.13.1)
       mini_portile2 (~> 2.7.0)
       racc (~> 1.4)
-    nokogiri (1.13.1-x86_64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.13.1-x86_64-linux)
-      racc (~> 1.4)
```

Comment:
I'm not sure how much to be concerned about this biting us, but I'm also happy to approve and let you run with it!